docs/news/index.qmd

```diff
@@ -9,6 +9,95 @@ format:
 
 This page provides the release notes associated with each release of RStudio and Posit Workbench. Please contact customer support (<a href="mailto:support@posit.co">support@posit.co</a>) for questions about the described changes.
 
+## RStudio 2023.06.0
+
+**"Mountain Hydrangea"**
+
+### New
+
+#### RStudio IDE
+- Updated to Electron 23.3.0 (#13030)
+- Moved Help panel font size setting to Appearance tab in Global Options (#12816)
+- Update openssl to 1.1.1t for Windows (rstudio/rstudio-pro#3675)
+- Improve visibility of focus rectangles on Server / Workbench Sign In page [Accessibility] (#12846)
```

Comment:
The same type of thing here. If we are going to use Updated, we should use Improved.

Or, we switch things over to "Improves, Adds, Updates, Fixes"